aten/src/ATen/native/mkldnn/MKLDNNConversions.cpp

```diff
@@ -508,6 +510,25 @@ static std::vector<Tensor> mkldnn_reorder_mkldnn_rnn_layer_weight(
   return {packed_w1, packed_w2};
 }
 
+static Tensor mkldnn_serialize(const Tensor& self) {
```

Comment:
is `get_mkldnn_serialized_md` more specific?

Reply:
Renamed the function